src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -287,7 +304,11 @@ public static boolean isCommonJsDynamicImportCallback(
    * Recognizes __webpack_require__ calls that are the .then callback of a __webpack_require__.e
    * call. Example:
    *
-   * <p>__webpack_require__.e(0).then(function() { return __webpack_require__(4); })
+   * <ul>
+   *   <li>__webpack_require__.e(0).then(function() { return __webpack_require__(4); })
+   *   <li>Promise.all([__webpack_require__.e(0)]).then(function() { return
```

Comment:
Is this new format just the new way that webpack4 translates
```
import("module").then(callback);
```
or are we supporting a new source pattern here?

More generally, do you have an idea of what our policy should be on doing more dynamic things like saving the promise to a variable or using import outside the global scope?